src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.vb

```diff
@@ -50,6 +81,32 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                 AssemblyVersionMajorTextBox, AssemblyVersionMinorTextBox, AssemblyVersionBuildTextBox, AssemblyVersionRevisionTextBox}
         End Sub
 
+        ''' <summary>
+        ''' This checks the properties for licenses to determine the initial state of the licensing section.
+        ''' Currently, If both are set, it will default to just enabling the <see cref="PackageLicenseExpression" />.
+        ''' In the future, it might be a good idea to have a warning and only output one value, but the nuget error after packing is helpful.
+        ''' <see cref="TryGetNonCommonPropertyValue" /> will get the property if it is set even if it is empty, but because the empty properties 
+        ''' are ignored elsewhere, they are ignored here.
+        ''' </summary>
+        Private Sub InitializeLicensing()
+            Dim PackageLicenseFileSet = TryCast(TryGetNonCommonPropertyValue(GetPropertyDescriptor("PackageLicenseFile")), String)
+            If (PackageLicenseFileSet IsNot Nothing AndAlso PackageLicenseFileSet IsNot "") Then
+                _newLicensePropertyDetectedAtInit = True
+                SetLicenseRadioButtons(False)
+            End If
+            Dim PackageLicenseExpressionSet = TryCast(TryGetNonCommonPropertyValue(GetPropertyDescriptor("PackageLicenseExpression")), String)
+            If (PackageLicenseExpressionSet IsNot Nothing AndAlso PackageLicenseExpressionSet IsNot "") Then
+                _newLicensePropertyDetectedAtInit = True
+                SetLicenseRadioButtons(True)
+            End If
+            Dim PackageLicenseUrlSet = TryCast(TryGetNonCommonPropertyValue(GetPropertyDescriptor("PackageLicenseUrl")), String)
+            If (PackageLicenseUrlSet IsNot Nothing AndAlso PackageLicenseUrlSet IsNot "") Then
+                LicenseLineLabel.BackColor = Drawing.SystemColors.Control
```

Comment:
What is this trying to do? Why are we changing the color?

Reply:
In some of the other property pages, the separators between sections is just an empty label with a ControlDark BackColor. I am using the same method here but I am also using it to display the deprecated warning for PackageLicenseUrl.